Content.Client/Access/UI/IdCardConsoleWindow.xaml.cs

```diff
@@ -1,4 +1,6 @@
 using System.Linq;
+using Content.Client.SS220.UserInterface.Controls; // ss220 add pin for ui
+using Content.Client.SS220.UserInterface.System.PinUI; //ss220 add pin for ui
```

Comment:
```suggestion
using Content.Client.SS220.UserInterface.Controls;
using Content.Client.SS220.UserInterface.System.PinUI; 
```
Там в пути уже 220, но и, честно говоря, помечать любые using лишнее